desktop/app/main-window.js

```diff
@@ -3,23 +3,17 @@ import {app, ipcMain} from 'electron'
 import menuHelper from './menu-helper'
 import resolveRoot from '../resolve-root'
 import hotPath from '../hot-path'
-import {showMainWindow} from '../shared/local-debug.desktop'
+import {allowLogin} from '../shared/util/feature-flags'
 
 export default function () {
   const mainWindow = new Window(
     resolveRoot(`renderer/index.html?src=${hotPath('index.bundle.js')}`), {
       width: 1600,
       height: 1200,
-      show: showMainWindow
```

Comment:
we have a feature flag to allow login now and not this debug flag for showing the main window